### ISSUE TITLE
rc15 stuck on first etcher client call to init USB drive partition table

### ISSUE THREAD

**Issue by rgaudin** [kiwix/kiwix-hotspot#312]
In GitLab by @Kelson42 on Nov 11, 2018, 08:54

If I run the etcher command from the command line (as administrator), same: it is stuck:

```
C:\Users\Kelson\AppData\Local\Temp\7zSC7CF.tmp\etcher-cli\etcher.exe -c -u -d \\.\PHYSICALDRIVE1
 C:\Users\Kelson\AppData\Local\Temp\7zSC7CF.tmp\mbr.img
```

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Nov 11, 2018, 08:59

OK. Could you extract the kiwix-hotpot exe (7z) and try again in an admin command line from a clean state ? ie: without a running/stuck etcher.

**Comment by rgaudin** [comment]
In GitLab by @Kelson42 on Nov 11, 2018, 09:12

Partition table was empty, so it seems #258 is not fixed. One time I have formated in FAT32 it goes through.

**Comment by rgaudin** [comment]
In GitLab by @Kelson42 on Nov 11, 2018, 09:38

@rgaudin same problem, stuck.

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Nov 11, 2018, 09:43

Thanks for testing. It really looks like #258 but it works for me even with an empty partition table so there might be an additional factor.

Could you verify (without rebooting) that it works if you create an partition on this card?

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Nov 11, 2018, 22:25

mentioned in commit cd5b73a408d86301cfe566a13af49f7d2e8625de

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Nov 11, 2018, 22:34

closed via commit cd5b73a408d86301cfe566a13af49f7d2e8625de
